packages/desktop-client/src/components/budget/MobileBudgetTable.js

```diff
@@ -416,16 +439,52 @@ class TotalsRow extends PureComponent {
             opacity: this.opacity,
           }}
         >
-          <CellValue
-            binding={rolloverBudget.groupBudgeted(group.id)}
-            style={{
-              ...styles.smallText,
-              width: 90,
-              fontWeight: '500',
-              textAlign: 'right',
-            }}
-            type="financial"
-          />
+          {!smallVPWidth ? (
```

Comment:
Similar thoughts as in [the block of ternaries above](https://github.com/actualbudget/actual/pull/1651/files#diff-60e094a44d38fac03fb722c1cf8eb9ddafcf42fbd2d15c0c9f4c3b7d1ec90474R294). Can you consolidate the logic & remove duplication of components?